NextcloudTalk/CallViewController.swift

```diff
@@ -852,9 +861,18 @@ class CallViewController: UIViewController,
                 // Notify callkit about room switch
                 self.delegate?.callViewController(self, wantsToSwitchFromRoom: self.room.token, toRoom: token)
 
+                // Store user that doesn't need to be notify in case we are switching from a one2one to an extended room
+                self.previousParticipants.removeAll()
+                if self.room.type == .oneToOne {
+                    self.previousParticipants.append(self.room.name)
+                }
+
                 // Assign new room as current room
                 self.room = newRoom
 
+                // Start call silently in new room
+                self.silentCall = true
```

Comment:
@Ivansss @SystemKeeper 
From what i understand, `silentCall` should be `false` here?
Because want that the call is ringing for all new participants, except for the `previousParticipants`, right?